recipe/meta.yaml

```diff
@@ -9,25 +9,36 @@ package:
 source:
   - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
     sha256: {{ sha256 }}
+    patches:
+      - win64_detect_conda.diff  # [win]
 
 build:
-  number: 1
-  skip: true  # [win or not linux64 or cuda_compiler_version in (undefined, "None")]
+  number: 0
+  # TODO(leofang): fix the Thrust compiling error (C3615) for Windows + CUDA 11.2
+  skip: true  # [(not win64 and not linux64) or cuda_compiler_version in (undefined, "None") or (win64 and cuda_compiler_version == "11.2")]
   script:
     # CuPy default detects CUDA from nvcc, but on Conda-Forge's dockers nvcc lives in a different place...
-    - export NVCC=$(which nvcc)
     # With conda-forge/nvcc-feedstock#58, CUDA_PATH is set correctly
-    - echo "nvcc is $NVCC, CUDA path is $CUDA_PATH"
+    - export NVCC=$(which nvcc)                                                  # [linux64]
+    - echo "nvcc is $NVCC, CUDA path is $CUDA_PATH"                              # [linux64]
+    - for /f "tokens=* usebackq" %%f in (`where nvcc`) do (set "dummy=%%f" && call set "NVCC=%%dummy:\=\\%%")  # [win]
+    - echo "nvcc is %NVCC%, CUDA path is %CUDA_PATH%"                                                          # [win]
 
     - {{ PYTHON }} -m pip install . --no-deps -vv
+    - if errorlevel 1 exit 1  # [win]
```

Comment:
Do we need this? Would think `conda-build` would handle this for us

Reply:
No, unfortunately haha. You'd think so, wouldn't you? This is so silly...
- https://github.com/cupy/cupy/blob/cd2550f188be264086ff0d0420d706f6e0460f5a/cupy/__init__.py#L11
- https://github.com/conda-forge/cudatoolkit-feedstock/blob/edee82fbd255c660b251a366e128eff26066e412/recipe/meta.yaml#L258

Reply:
Sorry, scratch the 1st link. I meant to link to my earlier https://github.com/conda-forge/cupy-feedstock/pull/113#issuecomment-806538078, but my browser is malfunctioning...

Reply:
All good that was my takeaway. Raised issue ( https://github.com/conda/conda-build/issues/4216 )

Reply:
Subscribed! Thanks John!